src/provisioner/provisioner.py

```diff
@@ -176,21 +172,27 @@ def cluster_bootstrap(cortx_conf_url: str = None):
 
         node_name = cortx_config_store.get(f'node>{node_id}>name')
 
-        Log.info(f'Starting cluster bootstrap on {node_id}:{node_name}')
+        return node_id, node_name
+
+    @staticmethod
+    def _provision_cluster(cortx_conf_url: str, mp_interfaces: list):
```

Comment:
Rename this to _provision_components as it calls component mini provisioner and not cluster.

Reply:
Done